core/src/main/java/org/apache/commons/jelly/JellyException.java

```diff
@@ -51,22 +45,19 @@ public JellyException(String message) {
     }
 
     public JellyException(String message, Throwable cause) {
-        super(message);
-        this.cause = cause;
+        super(message, cause);
     }
 
     public JellyException(Throwable cause) {
-        super(cause.getLocalizedMessage());
-        this.cause = cause;
+        super(cause);
```

Comment:
This changes behavior, see the adapted test. I don't feel strongly about this, so would be happy to restore the previous behavior.